.env.sample

```diff
@@ -1,3 +1,7 @@
+NAME=framework

```

Comment:
@oscarmild Change the name from #Framework to #cookbook-api

Reply:
Yes I will change.